archivers/unshield/Portfile

```diff
@@ -1,36 +1,35 @@
-PortSystem			1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name				unshield
-version				0.6
-categories			archivers compression sysutils
-license				MIT
-platforms			darwin
-maintainers			nomaintainer
-description			Extract files from InstallShield cabinet files
-
-long_description \
-	Extract files from InstallShield-style cabinet files (usually named \
-	data1.cab, data1.hdr, data2.cab, etc), versions 5 and later
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
 
+github.setup        twogood unshield 1.4.3
```

Comment:
please add the line
```
revision            0
```
underneath the `github.setup`

Reply:
Done!